Mathlib/Algebra/Notation/Defs.lean

```diff
@@ -216,6 +216,7 @@ end ite
 
 @[to_additive]
 class One (α : Type u) where
+  /-- The distinguished element `1` -/
```

Comment:
`Zero.zero` has the docstring `The zero element of the type.`
So maybe,
```suggestion
  /-- The unit element `1` of the type.  -/
```
to make the parallel more apparent?